### PR TITLE
Patch 2019-09-27: New tormentor class

### PR DESCRIPTION
**Pull request by Groovman**
### Class
* [x]  [Tormentor](https://gbf.wiki/Tormentor)

### Weapon
* [x]  [Devil's Finger](https://gbf.wiki/Devil%27s_Finger)
* [x]  [Dragon Buster](https://gbf.wiki/Dragon_Buster)

## Not implement
- [x] 執行Lv (Execution level)
- [x] Debuff success rate +10%
- [x] Boost to attack during break +15%

* Please merge in order of date.

### CODE REVIEW

File: src/global_const.js [hunk 1]
```diff
@@ -1461,6 +1461,18 @@ module.exports.Jobs = {
         "DaBonus": 9.0,
         "TaBonus": 6.0
     },
+    "torment": {
+        "name": "トーメンター",
+        "favArm1": "dagger",
+        "favArm2": "dagger",
+        "type": "pecu",
+        "atBonus": 3000.0,
```
Comment (kei-gbf) [inline src/global_const.js]: atBonus is 2000 ?

**Comment by kei-gbf** [comment]
Can we make Not implemented list? and put the list in PR comment

```md
## Not implement
- [ ] 執行Lv (Execution level)
- [ ] Debuff success rate +10%
- [ ] Boost to attack during break +15%
```

File: src/global_const.js (hunk 1 above)
Comment (Groovman) [reply src/global_const.js]: My bad read the wrong value from jp wiki. En wiki site wasn't up when I commited and pushed it.

**Comment by Groovman** [comment]
> Can we make Not implemented list? and put the list in PR comment
> 
> ```lisp
> ## Not implement
> - [ ] 執行Lv (Execution level)
> - [ ] Debuff success rate +10%
> - [ ] Boost to attack during break +15%
> ```

Done.

For execution level we would need class specific fields, someone could use that empty space under the class choice for it :)
Debuff success rate does not influence damage, so we might not need it, it might be useful information but for our case not that relevant, and all classes would need an extra field.
For break damage we would need a break damage state in the first place and would have to implement other things in addition, like break assassins buff/emps/ supports etc.

**Comment by kei-gbf** [comment]
- [ ] Dragon Buster lv100 2nd skill (main and Djeeta only)
  - additional damage on critical

**Comment by kei-gbf** [comment]
@Groovman 
can you update notice.js ChangeLog

**Comment by kei-gbf** [comment]
Notice ChangeLog update can be after Mirin update patch.

**Pull request merged**